website/content/docs/agent-and-proxy/agent/process-supervisor.mdx

```diff
@@ -6,7 +6,7 @@ description: >-
   into a process via environment variables using Consul Template markup.
 ---
 
-# Vault agent's process supervisor mode
+# Vault Agent's process supervisor mode
 
 Vault Agent's Process Supervisor Mode allows Vault secrets to be injected into
```

Comment:
The capitalization should match here with the above title

Reply:
Good catch! I'll update this